src/util/debug_module.js

```diff
@@ -289,7 +290,45 @@ class InternalDebugLogger {
         return tmp_mod.__level;
     }
 
+    message_format_without_color(level, args) {
```

Comment:
I do not think we need 2 functions, we can set the `level_color` in the orig function to be an empty string, and check if we want colors or not.

Reply:
done